corehq/apps/users/dbaccessors/all_commcare_users.py

```diff
@@ -51,3 +51,12 @@ def _clear_cache(doc):
         user.clear_quickcache_for_user()
     iter_bulk_delete(CommCareUser.get_db(), get_all_user_ids(), doc_callback=_clear_cache)
     User.objects.all().delete()
+
+
+def get_deleted_by_username(cls, username):
```

Comment:
again super nit picky but this might be clearer if it was `get_deleted_user_by_username`